security.rst

```diff
@@ -656,6 +656,94 @@ many other authenticators:
 
 .. _security-form-login:
 
+Login Programmatically
+----------------------
+
+.. versionadded:: 6.2
+
+    The :class:`Symfony\\Bundle\\SecurityBundle\\Security\\Security` class
+    was introduced in Symfony 6.2.
+    The :method:`Symfony\\Bundle\\SecurityBundle\\Security\\Security::login` method
+    was introduced in Symfony 6.2.
+
+You can log in an user programmatically using the `login()` method of the
+:class:`Symfony\\Bundle\\SecurityBundle\\Security\\Security` helper::
+
+
+use Symfony\Bundle\SecurityBundle\Security\Security;
+
+class ExampleController
+{
+    public function __construct(private Security $security)
+    {}
+
+    public function someMethod(): Response
+    {
+        $user = ... // Get the user to be authenticated
+
+        $this->security->login($user);
+
+        // Redirect the user to its account page for instance
+        // ...
+    }
+}
+
+The previous example only works if you have only one authenticator configured on the current firewall so if you have more than one authenticator, you need to specify which authenticator should be used.
+If no firewall is explicitly passed to the method, it will use the firewall from the current request. 
+And If the current request is not under any firewall, a ``\LogicException`` will be thrown.
```

Comment:
I think I would replace this paragraph with real code in the example above and add inline comments about when what is required.